### PR TITLE
WRR-1137: Updated package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by ion-andrusciac-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated package dependencies to fix security vulnerabilities

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Overrides changes:

- `axios: "^1.7.7"`
- `body-parser: "^1.20.3"`
- `path-to-regexp: "^0.1.10"`

### Links
[//]: # (Related issues, references)
WRR-1137

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac ion.andrusciac@lgepartner.com

### CODE REVIEW

File: sample-runner/agate/package.json [hunk 1]
```diff
@@ -30,12 +30,12 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.9.0",
-    "@enact/i18n": "^4.9.0",
+    "@enact/core": "^5.0.0-alpha.1",
```
Comment (daniel-stoian-lgp) [inline sample-runner/agate/package.json]: i don't think we should  update enact dependencies to an alpha version.
let's revert the entire file

File: sample-runner/core/package.json [hunk 2]
```diff
@@ -27,10 +27,10 @@
     "dist/*"
   ],
   "dependencies": {
-    "@enact/core": "^4.9.0",
-    "@enact/i18n": "^4.9.0",
-    "@enact/spotlight": "^4.9.0",
-    "@enact/ui": "^4.9.0",
+    "@enact/core": "^5.0.0-alpha.1",
```
Comment (daniel-stoian-lgp) [inline sample-runner/core/package.json]: i don't think we should  update enact dependencies to an alpha version.
let's revert the entire file

File: package.json [hunk 3]
```diff
@@ -69,7 +69,17 @@
     "typography-plugin-code": "^0.16.20"
   },
   "overrides": {
+    "@enact/docs-utils": {
```
Comment (daniel-stoian-lgp) [inline package.json]: my npm audit command ran locally on develop branch showed only 2 sources of high/critical vulnerabilities
1. lodash.template - whch cannot be resolved until gatsby-plugin-offline updates it's source code
2 "path-to-regexp":, which can be solved with an override
image

Can you please double check?

File: package.json [hunk 4]
```diff
@@ -69,7 +69,17 @@
...
+      "documentation": {
+        "vue-template-compiler": "^1.0.28"
```
Comment (daniel-stoian-lgp) [inline package.json]: this moderate vulnerability seems to come from @enact/docs-utils. Let's wait for a fix from there and not add this override here

File: package.json [hunk 5]
```diff
@@ -69,7 +69,17 @@
...
+      }
+    },
     "autolinker": "^4.0.0",
+    "axios": "^1.7.7",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "react-server-dom-webpack@0.0.0-experimental-c8b778b7f-20220825": {
+      "react": "^18.3.1"
+    },
```
Comment (daniel-stoian-lgp) [inline package.json]: i did not see any vulnerabilities related to these packages on local environment, after a deletion of node_modules and package-lock.json, on develop branch. Can you please double check?

File: package.json [hunk 6]
```diff
@@ -69,7 +69,17 @@
     "typography-plugin-code": "^0.16.20"
   },
   "overrides": {
+    "@enact/docs-utils": {
+      "documentation": {
+        "vue-template-compiler": "^1.0.28"
+      }
+    },
     "autolinker": "^4.0.0",
+    "axios": "^1.7.7",
```
Comment (daniel-stoian-lgp) [inline package.json:73]: i agree with this override :)

**Review by daniel-stoian-lgp — APPROVED** [review]

**Comment by hong6316** [comment]
- Please solve conflicts
- Add `overrides` changes on `Additional Considerations`

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**